cmd/preflight/cmd/check_container_test.go

```diff
@@ -23,7 +24,7 @@ var _ = Describe("Check Container Command", func() {
 	Context("when running the check container subcommand", func() {
 		Context("With all of the required parameters", func() {
 			It("should reach the core logic, but throw an error because of the placeholder values for the container image", func() {
-				_, err := executeCommand(checkContainerCmd(mockRunPreflight), "example.com/example/image:mytag")
+				_, err := executeCommand(checkContainerCmd(mockRunPreflightReturnNil), "example.com/example/image:mytag")
```

Comment:
These test suite adjustments, they're from a different PR, right?

Reply:
The changes are included from the original PR, I refactored them again to provide a positive and negative test. The original test never needed a negative test since it did not have an 
```
if err ! = nil
```
block of code. I hope this makes sense.